include/private/config/network_support_wrapper.h

```diff
@@ -0,0 +1,41 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/*
+ * Copyright 2021 Joel E. Anderson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __STUMPLESS_PRIVATE_CONFIG_WRAPPER_NETWORK_SUPPORT_H
+#  define __STUMPLESS_PRIVATE_CONFIG_WRAPPER_NETWORK_SUPPORT_H
+
+#  include <stumpless/config.h>
+
```

Comment:
You'll also need to move the preprocessor inclusion of the windows wrapper, as well as the `HAVE_SYS_SOCKET`/`HAVE_WINSOCK2` blocks, starting from line 25 in the original `include/private/config/wrapper.h` file to line 87.